rdrf/rdrf/importer.py

```diff
@@ -708,13 +708,11 @@ def _create_consent_sections(self, registry):
             for section_dict in self.data["consent_sections"]:
                 code = section_dict["code"]
                 section_label = section_dict["section_label"]
-                information_link = section_dict["information_link"]
-
                 section_model, created = ConsentSection.objects.get_or_create(
                     code=code, registry=registry, defaults={'section_label': section_label})
-                if not created:
-                    section_model.section_label = section_label
-                section_model.information_link = information_link
+                section_model.section_label = section_label
```

Comment:
Might be better to add a conditional check 

if "information_link" in section_dict:
     then set 

as earlier exported yaml files may not contain this key/value pair 

I've done this before as we'e introduced new model changes into the registry - the earlier exported yamls still need to imported